src/Client/Resources/keen-io-3_0.php

```diff
@@ -1,1252 +1,1021 @@
 <?php
 
+use \KeenIO\Client\Operations\Parameters\DefaultParameters;
+use \KeenIO\Client\Operations\Parameters\ParametersKey;
+use \KeenIO\Client\Operations\Parameters\ParameterKey;
+use \KeenIO\Client\Operations\OperationsKeys;
+use \KeenIO\Client\Operations\OperationKeys;
+use \KeenIO\Client\Operations\Parameters\ValueLocation;
+use \KeenIO\Client\Operations\Parameters\ValueHttpMethod;
+use \KeenIO\Client\Operations\Parameters\ValueType;
```

Comment:
I see a big drawback to this change: the array is now dynamic instead of being a static array that OPCache can keep in memory.

Reply:
btw, this refactoring should probably not be done in the same PR than adding support for access keys, as it makes it impossible to merge the new feature if the maintainers don't agree with the refactoring.

Reply:
yes I agree...
I will roll back...but it is really unmotivating to wait an response 9 months ;)

Reply:
note that I'm not a maintainer of this project. My review was a community review, not a maintainer review.